### DISCUSSION TITLE
Is gpth compatible with larger takeout file sizes?

### DISCUSSION THREAD

**Discussion by jvvlo** [TheLastGimbus/GooglePhotosTakeoutHelper#330] [I NEED HELP] [answered]
First of all, thank you for the great tool!

Walking through the tutorial images there is no comment on changing the file size. I assume this means that the default (2 GB) is optimal. Is that true? If so I am curious why?

I am downloading > 200 GB of data and I wonder what unforeseen problems may be encountered with takeout files of 50 GB.

**Comment by TheLastGimbus** [comment, marked as answer]
I've seen some weird memory problems at larger sizes, but i encurage you to try :+1: 

you should not fragment the takeout as gpth may not find links between files and you can end up with duplicates

**Reply by jvvlo** [reply]
Thank you!
